pkg/helpers/helm/helm.go

```
@@ -146,7 +149,7 @@ func (h *Helm) PrepareChart(repoName, repoURL string) error {
 		if err := f.WriteFile(repoFile, 0644); err != nil {
 			log.Fatal(err)
 		}
-		fmt.Printf("%q has been added to your repositories\n", repoName)
+		fmt.Fprintf(h.streams.Out, "%q has been added to your repositories\n", repoName)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)